data/spec-lanes.json

```diff
@@ -44,6 +55,90 @@
                         "description": "Traffic does not travel in this lane."
                     }
                 ]
+            },
+            "designated": {
+                "type": "string",
+                "description": "The predominant or designated traffic type, to hint at how to depict the lane. Should not be used to determine access restrictions.",
+                "anyOf": [
+                    {
+                        "const": "any",
+                        "description": "No designated type. Examples could be a living street or unmarked rural road."
+                    },
+                    {
+                        "const": "foot",
+                        "description": "Designated footpaths or sidewalks."
+                    },
+                    {
+                        "const": "bicycle",
+                        "description": "Designated cycleways."
+                    },
+                    {
+                        "const": "motor_vehicle",
+                        "description": "No significant pedestrian or bicycle usage, but may have busses, taxis, motorbikes, etc."
+                    },
+                    {
+                        "const": "bus"
+                    },
+                    {
+                        "const": "taxi"
+                    }
+                ]
+            },
+            "markings": {
+                "type": "array",
+                "description": "The road markings arranged from left to right.",
```

Comment:
Is this valid _only_ for `type=separator`? If not, what's an example?

Reply:
- [ ] valid only for type=separator